02-oop-inheritance/observerPattern.js

```diff
@@ -0,0 +1,140 @@
+//Inherit function
+var inherit = function inherit(child, parent){
+  var copyOfParent = Object.create(parent.prototype);
+  copyOfParent.constructor = child;
+ 	child.prototype = copyOfParent;	
+};
+
+//Augment function
+function augment(receivingClass, givingClass, methods) {
```

Comment:
This has no relation, try to understand what both functions are doing. I might think that you copied this function right away and dont know if you actually read what it does. The inherit functions is stored into a variable and the augment functions is not. Why? ...